test/ws/web/spec_expectations.clj

```diff
@@ -179,44 +179,80 @@
 
 (defexpect date-tests
   (expecting "String conversions"
-    (expect (jt/sql-date 1962 7 7) (->date "1962/7/7"))
-    (expect (jt/sql-date 1962 7 7) (->date "1962-7-7"))
-    (expect (jt/sql-date 1960 11 26) (->date "11/26/1960"))
-    (expect (jt/sql-date 1960 11 26) (->date "11-26-1960"))
-    ;; watch out for short year -- it will always be 2000+
-    (expect (jt/sql-date 2060 11 26) (->date "11/26/60")))
+             (expect (jt/sql-date 1962 7 7) (->date "1962/7/7"))
```

Comment:
Changing whitespace/indentation is a big no-no in Pull Requests since it makes it very hard to review just what was changed -- please revert all the whitespace changes.